pkg/registry/core/service/ipallocator/metrics.go

```diff
@@ -73,6 +74,17 @@ var (
 		},
 		[]string{"cidr", "scope"},
 	)
+	clusterIPAllocationLatency = metrics.NewHistogramVec(
```

Comment:
@dgrisonnet to review new metric

Reply:
I commented on 1/2 and the new metric looks good to me